backend/src/app.ts

```diff
@@ -69,21 +87,39 @@ app.post('/landlords', async (req, res) => {
   }
 });
 
+const isLandlord = (obj: LandlordWithId | ApartmentWithId): boolean => {
+  const keys = Object.keys(obj);
+  return keys.includes('contact');
```

Comment:
This function is `O(n)` which is unnecessary for just checking whether the object has a `contact` field or not. You can instead just do it in constant time using `if ('contact' in obj) { }`